atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -208,6 +209,36 @@ def build_list(self, manifests):
                 ],
         }, indent=4)
 
+    def check_existing_vr_tag(self):
+        self.primary_images = get_primary_images(self.workflow)
+
+        if not self.primary_images:
+            return
+
+        vr_image = None
+        for image in self.primary_images:
+            if '-' in image.tag:
```

Comment:
is this detection reliable? Checking just `-` seems too generic to me

Reply:
It is reliable for the moment, yes, and this method is used elsewhere in atomic-reactor (see image_image, koji_upload, and koji_import).

See OSBS-7169 for a better way of handling this in the future.